experimental/LieAlgebras/test/runtests.jl

```diff
@@ -7,3 +7,5 @@ include("LieAlgebraIdeal-test.jl")
 include("LieAlgebraHom-test.jl")
 include("LieAlgebraModule-test.jl")
 include("LieAlgebraModuleHom-test.jl")
+include("root_systems-test.jl")
+include("simple_lie_algebra-test.jl")
```

Comment:
```suggestion
```
gets already included in `LieAlgebra-test.jl` to have the conformance tests available